--- ./Makefile	2014-05-14 01:47:57.083489084 +0200
+++ ./Makefile	2014-05-14 02:02:41.419486536 +0200
@@ -31,21 +31,24 @@
 #
 
 # List of module objects that should be installed for all derivatives
-MODULES = memdisk/memdisk memdump/memdump.com modules/*.com \
-	com32/menu/*.c32 com32/modules/*.c32 com32/mboot/*.c32 \
-	com32/hdt/*.c32 com32/rosh/*.c32 com32/gfxboot/*.c32 \
-	com32/sysdump/*.c32 com32/lua/src/*.c32 com32/chain/*.c32
+#MODULES = memdisk/memdisk memdump/memdump.com modules/*.com \
+#	com32/menu/*.c32 com32/modules/*.c32 com32/mboot/*.c32 \
+#	com32/hdt/*.c32 com32/rosh/*.c32 com32/gfxboot/*.c32 \
+#	com32/sysdump/*.c32 com32/lua/src/*.c32 com32/chain/*.c32
+MODULES = 
 
 # syslinux.exe is BTARGET so as to not require everyone to have the
 # mingw suite installed
 BTARGET  = version.gen version.h version.mk
+#BOBJECTS = $(BTARGET) \
+#	mbr/*.bin \
+#	core/pxelinux.0 core/isolinux.bin core/isolinux-debug.bin \
+#	gpxe/gpxelinux.0 dos/syslinux.com \
+#	win32/syslinux.exe win64/syslinux64.exe \
+#	dosutil/*.com dosutil/*.sys \
+#	$(MODULES)
 BOBJECTS = $(BTARGET) \
-	mbr/*.bin \
-	core/pxelinux.0 core/isolinux.bin core/isolinux-debug.bin \
-	gpxe/gpxelinux.0 dos/syslinux.com \
-	win32/syslinux.exe win64/syslinux64.exe \
-	dosutil/*.com dosutil/*.sys \
-	$(MODULES)
+	mbr/mbr.bin
 
 # BSUBDIRs build the on-target binary components.
 # ISUBDIRs build the installer (host) components.
@@ -53,36 +56,44 @@
 # Note: libinstaller is both a BSUBDIR and an ISUBDIR.  It contains
 # files that depend only on the B phase, but may have to be regenerated
 # for "make installer".
-BSUBDIRS = codepage com32 lzo core memdisk modules mbr memdump gpxe sample \
-	   diag libinstaller dos win32 win64 dosutil
+#BSUBDIRS = codepage com32 lzo core memdisk modules mbr memdump gpxe sample \
+#	   diag libinstaller dos win32 win64 dosutil
+BSUBDIRS = core mbr libinstaller
 ITARGET  =
+#IOBJECTS = $(ITARGET) \
+#	utils/gethostip utils/isohybrid utils/mkdiskimage \
+#	mtools/syslinux linux/syslinux extlinux/extlinux
 IOBJECTS = $(ITARGET) \
-	utils/gethostip utils/isohybrid utils/mkdiskimage \
-	mtools/syslinux linux/syslinux extlinux/extlinux
-ISUBDIRS = libinstaller mtools linux extlinux utils
+	extlinux/extlinux
+#ISUBDIRS = libinstaller mtools linux extlinux utils
+ISUBDIRS = libinstaller extlinux
 
 # Things to install in /usr/bin
-INSTALL_BIN   =	mtools/syslinux
+#INSTALL_BIN   =	mtools/syslinux
 # Things to install in /sbin
 INSTALL_SBIN  = extlinux/extlinux
 # Things to install in /usr/lib/syslinux
-INSTALL_AUX   =	core/pxelinux.0 gpxe/gpxelinux.0 gpxe/gpxelinuxk.0 \
-		core/isolinux.bin core/isolinux-debug.bin \
-		dos/syslinux.com \
-		mbr/*.bin $(MODULES)
-INSTALL_AUX_OPT = win32/syslinux.exe win64/syslinux64.exe
-INSTALL_DIAG  =	diag/mbr/handoff.bin \
-		diag/geodsp/geodsp1s.img.xz diag/geodsp/geodspms.img.xz
+#INSTALL_AUX   =	core/pxelinux.0 gpxe/gpxelinux.0 gpxe/gpxelinuxk.0 \
+#		core/isolinux.bin core/isolinux-debug.bin \
+#		dos/syslinux.com \
+#		mbr/*.bin $(MODULES)
+INSTALL_AUX   =	mbr/mbr.bin $(MODULES)
+#INSTALL_AUX_OPT = win32/syslinux.exe win64/syslinux64.exe
+INSTALL_AUX_OPT = 
+#INSTALL_DIAG  =	diag/mbr/handoff.bin \
+#		diag/geodsp/geodsp1s.img.xz diag/geodsp/geodspms.img.xz
+INSTALL_DIAG  =
 
 # These directories manage their own installables
-INSTALLSUBDIRS = com32 utils dosutil
+#INSTALLSUBDIRS = com32 utils dosutil
+INSTALLSUBDIRS =
 
 # Things to install in /boot/extlinux
 EXTBOOTINSTALL = $(MODULES)
 
 # Things to install in /tftpboot
-NETINSTALLABLE = core/pxelinux.0 gpxe/gpxelinux.0 \
-		 $(MODULES)
+#NETINSTALLABLE = core/pxelinux.0 gpxe/gpxelinux.0 \
+#		 $(MODULES)
 
 all:
 	$(MAKE) all-local
@@ -113,32 +124,33 @@
 	$(PERL) version.pl $< $@ '< := @'
 
 local-install: installer
-	mkdir -m 755 -p $(INSTALLROOT)$(BINDIR)
-	install -m 755 -c $(INSTALL_BIN) $(INSTALLROOT)$(BINDIR)
+	#mkdir -m 755 -p $(INSTALLROOT)$(BINDIR)
+	#install -m 755 -c $(INSTALL_BIN) $(INSTALLROOT)$(BINDIR)
 	mkdir -m 755 -p $(INSTALLROOT)$(SBINDIR)
 	install -m 755 -c $(INSTALL_SBIN) $(INSTALLROOT)$(SBINDIR)
 	mkdir -m 755 -p $(INSTALLROOT)$(AUXDIR)
 	install -m 644 -c $(INSTALL_AUX) $(INSTALLROOT)$(AUXDIR)
-	-install -m 644 -c $(INSTALL_AUX_OPT) $(INSTALLROOT)$(AUXDIR)
-	mkdir -m 755 -p $(INSTALLROOT)$(DIAGDIR)
-	install -m 644 -c $(INSTALL_DIAG) $(INSTALLROOT)$(DIAGDIR)
-	mkdir -m 755 -p $(INSTALLROOT)$(MANDIR)/man1
-	install -m 644 -c man/*.1 $(INSTALLROOT)$(MANDIR)/man1
-	: mkdir -m 755 -p $(INSTALLROOT)$(MANDIR)/man8
-	: install -m 644 -c man/*.8 $(INSTALLROOT)$(MANDIR)/man8
+	#-install -m 644 -c $(INSTALL_AUX_OPT) $(INSTALLROOT)$(AUXDIR)
+	#mkdir -m 755 -p $(INSTALLROOT)$(DIAGDIR)
+	#install -m 644 -c $(INSTALL_DIAG) $(INSTALLROOT)$(DIAGDIR)
+	#mkdir -m 755 -p $(INSTALLROOT)$(MANDIR)/man1
+	#install -m 644 -c man/*.1 $(INSTALLROOT)$(MANDIR)/man1
+	#: mkdir -m 755 -p $(INSTALLROOT)$(MANDIR)/man8
+	#: install -m 644 -c man/*.8 $(INSTALLROOT)$(MANDIR)/man8
 
 install: local-install
 	set -e ; for i in $(INSTALLSUBDIRS) ; do $(MAKE) -C $$i $@ ; done
 
-netinstall: installer
-	mkdir -p $(INSTALLROOT)$(TFTPBOOT)
-	install -m 644 $(NETINSTALLABLE) $(INSTALLROOT)$(TFTPBOOT)
+#netinstall: installer
+#	mkdir -p $(INSTALLROOT)$(TFTPBOOT)
+#	install -m 644 $(NETINSTALLABLE) $(INSTALLROOT)$(TFTPBOOT)
 
 extbootinstall: installer
 	mkdir -m 755 -p $(INSTALLROOT)$(EXTLINUXDIR)
 	install -m 644 $(EXTBOOTINSTALL) $(INSTALLROOT)$(EXTLINUXDIR)
 
-install-all: install netinstall extbootinstall
+#install-all: install netinstall extbootinstall
+install-all: install extbootinstall
 
 local-tidy:
 	rm -f *.o *.elf *_bin.c stupid.* patch.offset
